src/metalnx-web/src/main/java/com/emc/metalnx/controller/BrowseController.java

```diff
@@ -578,25 +561,10 @@ public String modifyAction(@ModelAttribute final CollectionOrDataObjectForm coll
         // checking if the previousPath collection/dataobject was marked as favorite:
         String username = irodsServices.getCurrentUser();
         String zoneName = irodsServices.getCurrentUserZone();
-        DataGridUser user = userService.findByUsernameAndAdditionalInfo(username, zoneName);
-        boolean isMarkedFavorite = favoritesService.isPathFavoriteForUser(user, previousPath);
-        logger.info("Favorite status for previousPath: " + previousPath + " is: " + String.valueOf(isMarkedFavorite));
-
+        DataGridUser user = userService.findByUsernameAndZone(username, zoneName);
```

Comment:
Codacy says the variable `user` is unused - maybe we can just remove it.

Reply:
Yes, and that leads to the previous two lines being unnecessary.  All of these are for favorites which are no longer supported so it makes sense.  Removing these three lines plus the leading comment.

Reply:
done